README.md

```diff
@@ -10,9 +10,9 @@ Feel free to file issues or ask questions on our [issue tracker](http://github.c
 
 ![Build status for master branch](https://mseng.visualstudio.com/_apis/public/build/definitions/b21de0ba-dffd-474c-b888-7112d5e6397b/2921/badge)
```

Comment:
This badge doesn't provide any valuable information to the end user, what do you think about removing it? Users expect the master branch to be successfully built. If it's not, it's a bad impression and we're publicly advertising it.

Reply:
This isn't what I check to see if the build is passing/failing, so I don't personally care either way. If our build is not passing and I don't notice, and a user lets me know, I'll be happy about it.

It's a bit of advertising for Azure DevOps... it would be more useful if the build was public and you could navigate to it via the badge.

Reply:
I understand the logic behind azure advertisement, but it's kind of useless right now because it just leads to a page with a picture. How about making it point [here](https://azure.microsoft.com/en-us/services/devops/)? 

The public build is a better--if there's no easy way to do it right now, we can consider it down the line.

Reply:
I understand the logic behind azure advertisement, but it's kind of useless right now because it just leads to a page with a picture. How about making it point [here](https://azure.microsoft.com/en-us/services/devops/)?

The public build is a better--if there's no easy way to do it right now, we can consider it down the line.

Reply:
I've removed it. We can add it back if/when it ever provides value.